tests/common/quantization/test_passes.py

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) 2023 Intel Corporation
```

Comment:
Before we put such tests in `tests/post_training`

Reply:
Can you put the tests in `tests/post_training` because passes for quantization are used only in `MinMax` (PTQ)?